mitxpro/utils.py

```diff
@@ -89,6 +89,20 @@ def format_datetime_for_filename(datetime_object, include_time=False, include_ms
     return datetime_object.strftime("_".join(format_parts))
 
 
+def case_insensitive_equal(str1, str2):
```

Comment:
should we add an exception for the case where the provided arguments are not strings?

Reply:
I think it's OK in this case to trust clients of this function to provide strings